.crontab/ps1/aws_tokens

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+aws_tokens() {
+  local -r now=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+  local -r _aws_profiles=($@)
+  local token_expiration=""
+  local out=""
+
+  for aws_profile in "${_aws_profiles[@]}"; do
+    token_expiration="$(aws configure get profile.${aws_profile}.x_security_token_expires)"
+    if [[ "${token_expiration}" && $(datediff "${now}" "${token_expiration}" -f '%S') -gt 0 ]]; then
+      out="${out} [${aws_profile} $(datediff "${now}" "${token_expiration}" -f '%Mm')]"
+    fi
+  done
+
+  echo ${out}
```

Comment:
Consider wrapping the variable in quotes
```suggestion
  echo "${out}"
```
See [this article](https://nickjanetakis.com/blog/here-is-why-you-should-quote-your-variables-in-bash) as to why